CONTRIBUTING.md

```diff
@@ -115,6 +116,27 @@ the project with Local File System input.
 resources/sample_md_reflections/lab1, resources/sample_md_reflections/lab2, resources/sample_md_reflections/lab3
 ```
 
+#### Missing Module Gensim or gensim.summarization With Windows
```

Comment:
I think it might be a better idea to use a more generic header, maybe "Develop GatorMiner on Windows". That way, any future Windows setup can be added to this section.